TramsDataApi.Test/Factories/TrustResponseFactoryTests.cs

```diff
@@ -93,337 +93,13 @@ public void TrustResponseFactory_CreatesTrustResponse_FromAGroup_WhenIfdTrustDat
                 },
                 Ukprn = group.Ukprn
             };
-            var academyResponses = new List<AcademyResponse>();
+            var academyResponses = new List<EstablishmentResponse>();
```

Comment:
I'd rename the var too, just let intellisense do the lifting

Reply:
Doing to also rename the response object, as it's currently `Academies`, which isn't quite accurate as it's any establishment